ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+  that cause the EJB remote invocation proceeds to other two servers. Then examine the behaviour.
+
+=== Running the quickstart on OpenShift
+
+. <<_running_on_openshift_start_postgresql_database, Start the PostgreSQL database>> in OpenShift.
+. <<_running_on_openshift_build_the_application, Use OpenShift `oc` commands to build this quickstart>>, and deploy it to the OpenShift registry.
+. <<_running_on_openshift_install_productname_operator, Install the {productName} Operator>> to OpenShift.
+. <<_running_on_openshift_run_the_quickstart_with_productname_operator, Deploy the `CustomerResource` definition of this quickstart>> to OpenShift
+  and setup OpenShift permissions to form the {productName} cluster.
+. <<_running_on_openshift_verify_the_quickstarts, Run the HTTP calls against `server1`>>
+  that cause the EJB remote invocation proceeds to other two servers. Then examine the behaviour.
+
+// System Requirements
+include::../shared-doc/system-requirements.adoc[leveloffset=+1]
+// Use of {jbossHomeName}_1 and {jbossHomeName}_2
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+// Install the Quickstart Parent Artifact in Maven
+include::../shared-doc/install-quickstart-parent-artifact.adoc[leveloffset=+1]
+
+[#_setup_the_environment_]
+== Setup the environment
+
+You must set up all servers to use the EJB remote calls.
+
+It's needed to start three instances of an application server.
+The easiest way to do so on local computer is to copy the {productName} installation directory
+to three separate directories.
+
+The first server (`server1`) is a sender &ndash; it contains an EJB that executes
+a remote call to one of the remote servers (`server2`,`server3`), which are receivers.
+The installation directory for the `server1` is referred as `__{jbossHomeName}_1__`,
+for the `server2` it's `__{jbossHomeName}_2__` and for the `server3` it's `__{jbossHomeName}_3__`.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+# considering the ${jbossHomeName} is installation directory of the {productName}
+cp -r ${jbossHomeName} server1
+{jbossHomeName}_1="$PWD/server1"
+cp -r ${jbossHomeName} server2
+{jbossHomeName}_2="$PWD/server2"
+cp -r ${jbossHomeName} server3
+{jbossHomeName}_3="$PWD/server3"
+----
+
+[#_configure_the_credentials_for_remote_call_authentication]
+=== Configure the credentials for remote call authentication
+
+For the remote call could be processed successfully from `server1` to either `server2`
+or to `server3` there has to be created a user at the receiver side
+which the call will be authenticated with.
+
+Use the following procedures for `server2` and `server3`.
+Run the procedure for directories
+`__{jbossHomeName}_2__` and `__{jbossHomeName}_3__`.
+
+[#add_application_user]
+// Add the Authorized Application User
+include::../shared-doc/add-application-user.adoc[leveloffset=+3]
+
+[NOTE]
+====
+For the `add-user.sh` (or `.bat`) command you may add a parameter `-ds`.
```

Comment:
```suggestion
For the `add-user.sh` (or `.bat`) command you, can add the parameter `-ds`.
```
Our style guide recommends using "can" instead of "may".  "May" is ambiguous, as it might mean capability or permission. "Can" is clearer.